framework/src/state_machine/types.ts

```diff
@@ -93,7 +93,7 @@ export interface CommandVerifyContext<T = undefined> {
 	chainID: Buffer;
 	transaction: Transaction; // without decoding params
 	params: T;
-	getMethodContext: () => ImmutableMethodContext;
+	getMethodContext: () => MethodContext;
```

Comment:
this should be `ImmutableMethodContext` because it cannot use `set` in verify context